Sources/OpenTelemetryApi/Baggage/Propagation/W3CBaggagePropagator.swift

```diff
@@ -20,54 +20,78 @@ public struct W3CBaggagePropagator: TextMapBaggagePropagator {
 
   static let headerBaggage = "baggage"
 
+  private func isValidKeyValuePair(_ keyValue: String) -> (key: String, value: String)? {
+    let parts = keyValue.split(separator: "=", maxSplits: 1)
+    guard parts.count == 2 else { return nil }
+
+    return (String(parts[0]), String(parts[1]))
+  }
+
+  private func percentEncode(_ value: String) -> String {
```

Comment:
is this method used anywhere?

Reply:
Good catch! It was used during the development, but no longer need this. Just removed it.